src/Trie.mo

```diff
@@ -733,6 +743,41 @@ module {
     )
   };
 
+  /// Returns an `Iter` over the key-value entries of the trie.
+  ///
+  /// Each iterator gets a _persistent view_ of the mapping, independent of concurrent updates to the iterated map.
+  public func iter<K, V>(t : Trie<K, V>) : I.Iter<(K, V)> {
```

Comment:
indentation is inconsistent with rest of file.

Reply:
I agree it looks strange, but I'm using the emacs mode's choices consistently everywhere.

I think there are two things that make it look strangely (over) indented here:
 - `object` around the entire thing, to create the `Iter`
 - double `switch` statement, the "best" syntax for showing off the emacs mode's worst behavior.
 
 I can manually remove the double-indentation for the inner `switch`, but what else would you change?

Reply:
Committed your format suggestions.  Thanks!